twisted_tools/buildbot.py

```diff
@@ -3,6 +3,7 @@
 import treq
 import twisted
 
+BASE_URL = 'http://buildbot.twistedmatrix.com'
```

Comment:
Of course. Everyone is watched by intelligence agencies at all times. See for example https://glyph.twistedmatrix.com/2015/01/security-as-stencil.html :)